app/models/form/sales/pages/organisation.rb

```diff
@@ -10,24 +10,40 @@ def questions
     ]
   end
 
-  def routed_to?(_log, current_user)
-    if FeatureToggle.merge_organisations_enabled?
+  def routed_to?(log, current_user)
+    return false unless current_user
+    return true if current_user.support?
+    return false unless FeatureToggle.sales_managing_organisation_enabled?
 
-      return false unless current_user
-      return true if current_user.support?
+    return true if stock_owners_with_own_stock_count(current_user) > 1
+    return true if current_user.organisation.holds_own_stock? && stock_owners_with_own_stock_count(current_user) >= 1
```

Comment:
can we combine this and the previous line with a new method like `manages_logs_for_multiple_stock_owners?(current_user)` for clarity?

Reply:
Sure!